ui/src/shared/components/EmptyGraphError.tsx

```diff
@@ -0,0 +1,63 @@
+// Libraries
+import React, {useState, FunctionComponent} from 'react'
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+// Components
+import {
+  Button,
+  ComponentSize,
+  ComponentColor,
+  IconFont,
+} from '@influxdata/clockface'
+import FancyScrollbar from 'src/shared/components/fancy_scrollbar/FancyScrollbar'
+
+interface Props {
+  message: string
+  testID?: string
+}
+
+const EmptyGraphError: FunctionComponent<Props> = ({message, testID}) => {
+  const [didCopy, setDidCopy] = useState(false)
+
+  const buttonText = didCopy ? 'Copied!' : 'Copy'
+  const buttonColor = didCopy ? ComponentColor.Success : ComponentColor.Default
+
+  const onClick = () => {
+    setDidCopy(true)
+    setTimeout(() => setDidCopy(false), 2000)
+  }
+
+  return (
+    <div className="cell--view-empty" data-testid={testID}>
+      <div className="empty-graph-error">
+        <CopyToClipboard text={message}>
+          <Button
```

Comment:
Speak with @ischolten about standardizing copy to clipboard implementation across the application.

Reply:
Will do. I first tried using the `CodeSnippet` component used in the Telegraf configuration screen. The always-visible big purple "Copy to Clipboard" button felt out of place on the dashboard, which is why I created yet another way of doing the same thing :|